src/routes/index.js

```diff
@@ -1,11 +1,13 @@
 import express from 'express';
 import user from './api/user.routes.js';
 import product from './api/products.routes.js';
+import updateUserPassword from './api/updatePassword.routes';
 
 
 const routes = express.Router();
 routes.use('/users', user);
 routes.use('/products', product);
+routes.use('/users', updateUserPassword);
```

Comment:
This is duplication of 
routes.use('/users', updateUserPassword);
where this must be appended to routes.use('/users', user);